filebeat/module/system/syslog/ingest/pipeline.json

```diff
@@ -1,41 +1,48 @@
 {
-	"description": "Pipeline for parsing Syslog messages.",
-	"processors": [
-		{
-			"grok": {
-				"field": "message",
-				"patterns": [
-					"%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{SYSLOGHOST:system.syslog.hostname} %{DATA:system.syslog.program}(?:\\[%{POSINT:system.syslog.pid}\\])?: %{GREEDYMULTILINE:system.syslog.message}",
-					"%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{GREEDYMULTILINE:system.syslog.message}"
-				],
-        "pattern_definitions" : {
-          "GREEDYMULTILINE" : "(.|\n)*"
+    "description": "Pipeline for parsing Syslog messages.",
+    "processors": [
+        {
+            "grok": {
+                "field": "message",
+                "patterns": [
+                    "%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{SYSLOGHOST:host.hostname} %{DATA:process.name}(?:\\[%{POSINT:process.pid:int}\\])?: %{GREEDYMULTILINE:system.syslog.message}",
+                    "%{SYSLOGTIMESTAMP:system.syslog.timestamp} %{GREEDYMULTILINE:system.syslog.message}"
+                ],
+                "pattern_definitions" : {
+                    "GREEDYMULTILINE" : "(.|\n)*"
+                },
+                "ignore_missing": true
+            }
         },
-        "ignore_missing": true
-			}
-    },
-		{
-      "remove": {
-        "field": "message"
-      }
-    },
-    {
-      "date": {
-        "field": "system.syslog.timestamp",
-        "target_field": "@timestamp",
-        "formats": [
-					"MMM  d HH:mm:ss",
-					"MMM dd HH:mm:ss"
-        ],
-        {< if .convert_timezone >}"timezone": "{{ beat.timezone }}",{< end >}
-        "ignore_failure": true
-      }
-    }
-	],
-  "on_failure" : [{
-    "set" : {
-      "field" : "error.message",
-      "value" : "{{ _ingest.on_failure_message }}"
-    }
-  }]
+        {
+            "remove": {
+                "field": "message"
+            }
+        },
+        {
+            "rename": {
+                "field": "system.syslog.message",
+                "target_field": "message",
+                "ignore_missing": true
+            }
+        },
+        {
+            "date": {
+                "field": "system.syslog.timestamp",
```

Comment:
Do we need to remove the `system.syslog.timestamp` field (follow up PR)?

Reply:
That was one of my questions in the Caveats section.

Actually, given that #8083 is picking up steam again, I'd say we're good to delete this field. I'll do it in this PR.

If someone cares about seeing anything about their original message (not just the timestamp), they should preserve the whole original message.